src/rules/DEV/JS-TS/FRONT/nginx/index.ts

```diff
@@ -1,48 +1,46 @@
 import { YesNo } from './../../../../../choice/index';
-import { WriteFileError } from './../../../../../errors/FileErrors';
-import { DirError } from './../../../../../errors/DirErrors';
-import { RuleRegister } from '../../../../rule-register';
 import { StackRegister } from '../../../../../stacks/stack-register';
 import Angular from '../../../../../stacks/angular';
 import { React } from '../../../../../stacks/react';
 import VueJS from '../../../../../stacks/vue-js';
-import { ensureDir, copy } from 'fs-extra';
+import { ensureDir } from 'fs-extra';
 import Globals from '../../../../../utils/globals';
 import { logger } from '../../../../../logger';
 import Choice from '../../../../../choice';
-import { promptForRule } from '../../../../../utils/prompt';
-import Dockerfile from '../dockerfile';
+import { myCopy } from '../../../../../utils/file-utils';
 
-@RuleRegister.register
 @StackRegister.registerRuleForStacks([Angular, React, VueJS])
 export default class Nginx {
-  private configDirPath = Globals.rootPath + 'config';
+  private configDirPath = Globals.rootPath + 'nginx-config';
   private configFilePath = this.configDirPath + '/nginx.conf';
+  private defaultConfFilePath = __dirname + '/nginx.conf';
 
   async shouldBeApplied(): Promise<boolean> {
     return true;
   }
 
   async apply(apply: boolean): Promise<void> {
     if (apply) {
-      return ensureDir(this.configDirPath)
-        .then(
-          () => {
-            return copy(__dirname + 'nginx.conf', this.configFilePath);
-          },
-          err => {
-            throw new DirError(err, this.configDirPath, this.getName());
-          },
-        )
-        .then(
-          () => {
-            logger.info('Succesfully copied nginx.conf file to config folder.');
-            return promptForRule(new Dockerfile());
-          },
-          err => {
-            throw new WriteFileError(err, this.configFilePath, this.getName());
-          },
-        );
+      return new Promise((resolve, reject) => {
+        ensureDir(this.configDirPath)
+          .then(() => {
+            return myCopy(this.defaultConfFilePath, this.configFilePath);
+          })
+          .then(
+            () => {
+              logger.info(
+                'Succesfully copied nginx.conf file to config folder.',
+              );
+              resolve();
+            },
+            err => {
+              if (err.message.endsWith('already exists')) {
```

Comment:
We can maybe use error code instead of a `endsWith` ?

Reply:
That's what I wanted to do but sadly there is no error code in the thrown error...